api/wait

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```

Comment:
Maybe this file should be named `wait.sh`, not have a shebang line and instead host a function `wait_for_deps` which you can call from the entrypoint scripts (after sourcing). This would look a bit cleaner, and standalone execution of this library-like functionality would not be possible.

Reply:
What's the benefit of doing so?

In a similar case, the `apiwait` script in the e2e test container is called directly from `.travis.yml`. In that case, your proposed changes would yield to additional refactoring there.

Reply:
I don't like having an executable for what's really a library function, but I don't feel strongly. It's ok the way it is.